cmd/talosctl/cmd/talos/upgrade.go

```diff
@@ -138,7 +144,7 @@ func upgradeGetActorID(ctx context.Context, c *client.Client) (string, error) {
 }
 
 func init() {
-	upgradeCmd.Flags().StringVarP(&upgradeCmdFlags.upgradeImage, "image", "i", "", "the container image to use for performing the install")
+	upgradeCmd.Flags().StringVarP(&upgradeCmdFlags.upgradeImage, "image", "i", fmt.Sprintf("%s:%s", constants.DefaultInstallerImage, version.Tag), "the container image to use for performing the install")
```

Comment:
this can be used instead:
```suggestion
	upgradeCmd.Flags().StringVarP(&upgradeCmdFlags.upgradeImage, "image", "i", fmt.Sprintf("%s/%s/installer:%s", gendata.ImagesRegistry, gendata.ImagesUsername, gendata.VersionTag), "the container image to use for performing the install")
```

gendata coming from: `"github.com/siderolabs/talos/pkg/machinery/gendata"`

Reply:
Ah, nice!